### ISSUE TITLE
no-global-jquery flags a pattern incorrectly

### ISSUE THREAD

**Issue by efx** [ember-cli/eslint-plugin-ember#211]
no-global-jquery errors based on the following snippet:
```js
function firstRowText($) {
	return $('table tbody tr td:first-child').text().replace(/\s/g, '');
}
```
This is not a global instance, so the rule should not error here.
I see 2 quick workarounds:
1. rename the function argument to be something else
2. add `//eslint-disable-next-line ember/no-global-jquery` before the line

**Comment by rwjblue** [comment]
I'd prefer:

3. fix the rule 😸

**Comment by rwjblue** [comment]
Good catch, thank you for reporting!

**Comment by efx** [comment]
anytime!

**Comment by bmish** [comment]
Fixed by #1063.
